doc-src/admin-guide/admin-guide-single.adoc

```diff
@@ -45,3 +45,5 @@ include::{includedir}/support_and_troubleshooting.adoc[]
 
 include::{includedir}/opensource_technologies.adoc[]
 
+include::{includedir}/security_configuration.adoc[]
```

Comment:
@efeubiqube 
this doc needs to be included after include::{includedir}/ha_installation.adoc[]